### DISCUSSION TITLE
View Current Standards

### DISCUSSION THREAD

**Discussion by steven-techtron** [KelvinTegelaar/CIPP#1111] [Ideas]
It would be nice to be able to run a check on the current standards to show what is currently enabled so when you enable items you will know if their will be a change or not.  Its also a nice way to see what's currently enabled.

Also its not clear from the GUI, when running the standards if set to off, will it leave the current settings as is, or will it disable them?

**Comment by KelvinTegelaar** [comment]
Most standards show up in the bpa.  You can see which standards you have enabled from the list and edit standards screen. Your second question is in the docs; standards only change when you enable things. Nothing gets disabled.
